app/lib/core/models/transporter_info.dart

```diff
@@ -47,4 +47,23 @@ class TransporterInfo {
         trainingExpiryDate = trainingExpiryDate;
 
   List<String> driverNames() => List.unmodifiable(_driverNames);
+
+  Map<String, dynamic> toMap() {
```

Comment:
Looks like we'll be reusing this in almost every ATR-related object.

- [ ] Could we use a library like [this one](https://api.dart.dev/stable/2.8.4/dart-convert/dart-convert-library.html) to make all the `toMap`? 

What worries me about all these functions is that we could miss a field in them, like if you missed `driverNames` below, there's no way you would know immediately until a test or something failed. I read a little on dart forums and it looks like there's no automatic `toMap` for objects yet.